src/app/components/fluid-grid/index.html

```diff
@@ -9,6 +9,34 @@
     The fluid grid component provides a responsive 12-column layout to organize content for all device sizes. The fluid grid consists of the <sky-code>sky-fluid-grid</sky-code>, <sky-code>sky-row</sky-code>, and <sky-code>sky-column</sky-code> tags. You place <sky-code>sky-column</sky-code> elements in <sky-code>sky-row</sky-code> elements, and then you can specify how much of the layout to use for each <sky-code>sky-column</sky-code> element on extra-small, small, medium, and large screens. The <sky-code>sky-fluid-grid</sky-code> parent element ensures consistent styling for all fluid grid instances.
   </sky-demo-page-summary>
 
+  <sky-demo-page-properties sectionHeading="Fluid grid properties">
+    <sky-demo-page-property
+      propertyName="disableMargin"
+      defaultValue="false"
+      isOptional="true"
+    >
+      Disables the outer left and right margin of the fluid grid container. This property accepts <sky-code>boolean</sky-code> values.
```

Comment:
Do we need "outer" here? Is there an "inner" margin that we are distinguishing from? ... Also, should "margins" be plural? This reads to me like a left margin and a right margin, which suggests it should be plural. But I'm not sure.

Reply:
Yeah plural is fine. I think I added "outer" just to distinguish the difference between this and the gutters. I'm not sure if there is some better standardized language around these parts of the grid that would be a better fit?

Reply:
I updated this to make "margins" plural. ... As for "outer," I'd just leave it. I don't know of any better standardized language. I just wasn't sure if we needed "outer" given that it seemed like the end of the sentence makes it clear that we are referring to the margin of the container, not the individual columns. And since `gutterSize` description sounds like that padding is only for in between columns (as opposed to the left of the leftmost column and the right of the rightmost column), I thought "outer" might be extraneous. But better to err on the side of leaving it in for clarity than deleting it to tighten up.